File: DotNet/SETPractice/SOLIDPrinciples/SingleResponsibility/EmailService.cs
using System.Net.Mail;

namespace SOLIDPrinciples.SingleResponsibility
{
    public class EmailService
    {
        SmtpClient smtpClient;
        public EmailService() 
        { 
            smtpClient = new SmtpClient();
        }

        public bool ValidateEmail(string email)
        {
            return email.Contains("@");
        }

        public void SendMessage(MailMessage message)
        {
            // smtpClient.Send(message);
            Console.WriteLine("Message Delivered!");
        }
    }
}

File: Java/SETPractice/src/main/java/SOLIDPrinciple/SingleResponsibility/EmailService.java
package SOLIDPrinciple.SingleResponsibility;

public class EmailService {

    public boolean ValidateEmail(String email) {
        return email.contains("@");
    }

    public void SendEmail(String message) {
        System.out.println(message);
        System.out.println("Email delivered successfully!");
    }
}
